conda/core/index.py

```diff
@@ -40,21 +53,31 @@ def check_allowlist(channel_urls):
 
 @time_recorder("get_index")
 def get_index(
-    channel_urls=(),
-    prepend=True,
-    platform=None,
-    use_local=False,
-    use_cache=False,
-    unknown=None,
-    prefix=None,
-    repodata_fn=context.repodata_fns[-1],
-):
+    channel_urls: list[str] = (),
```

Comment:
```suggestion
    channel_urls: tuple[str] = (),
```

The default argument is a tuple, so we should assume this argument is always a tuple.